app/ChooseAccount.php

```diff
@@ -103,7 +114,6 @@ function () {
                     'automate' => $automate
```

Comment:
Maybe you could rename the (older) "automate" concept here to something like "auto_submit_form_via_js" to avoid any confusion with the new "automate_without_js" concept?